src/extremes_based/extremesdim.jl

```diff
@@ -34,28 +71,21 @@ function extremevaltheory_dims(X, p; kw...)
 end
 
 """
-    extremevaltheory_dims_persistences(x::AbstractStateSpaceSet, p::Real; kwargs...)
+    extremevaltheory_dims_persistences(x::AbstractStateSpaceSet, p::type; kwargs...)
 
 Return the local dimensions `Δloc` and the persistences `θloc` for each point in the
-given set for quantile probability `p`, according to the estimation done via extreme value
-theory [Lucarini2016](@cite).
+given set. The type `p` tells the function which approach to use when computing the
+dimension, see `BlockMaxima` and `Exceedances`. The exceedances approach follows the
```

Comment:
Instead of type `Name` , wrap this in an `@docs` reference block. Wrap the quoted name `Name` in [ ] and in `(@ref)`.

Reply:
After this sentence you should also add the sentence "providing `p::Real` defaults to using the exceedances method".